clients/python/girder_client/__init__.py

```diff
@@ -484,10 +489,10 @@ def sendRestRequest(self, method, path, parameters=None,
                 return result.json()
             else:
                 return result
-        # TODO handle 300-level status (follow redirect?)
```

Comment:
Ah, I didn't see your commit messages. It's becoming rare for people to include useful information in them.